service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java

```diff
@@ -277,4 +278,30 @@ public static JudgeAndLegalAdvisor createJudgeAndLegalAdvisor(final String legal
             .judgeTitle(judgeTitle)
             .build();
     }
+
+    public static List<Element<Direction>> createDirection(DirectionAssignee assignee) {
+        return ImmutableList.of(
+            Element.<Direction>builder()
+                .value(Direction.builder()
+                    .directionText("Mock direction text")
+                    .assignee(assignee)
+                    .build())
+                .build());
+    }
+
+    public static List<Element<Direction>> createCustomDirection(DirectionAssignee assignee) {
+        return ImmutableList.of(
+            Element.<Direction>builder()
+                .value(Direction.builder()
+                    .directionText("Mock direction text")
+                    .assignee(assignee)
+                    .readOnly("No")
+                    .custom("Yes")
+                    .build())
+                .build());
+    }
+
+    public static List<Element<Direction>> createUnassignedDirection() {
+        return createDirection(null);
+    }
```

Comment:
I realllyyyyy don't like this caseDataGeneratorHelper being in our main source code. A problem for another time. Code for tests should exist in test directories